projects/amsui/src/lib/solr-service/solr.service.ts

```diff
@@ -33,6 +33,7 @@ export abstract class SolrService implements OnDestroy {
   protected searchResultsSubscription!: Subscription;
   protected solrSearchURL = '';
   protected solrSuggestURL = '';
+  protected operator: string = 'AND';
```

Comment:
It's best practice in TS that if you already add a simple value (like string) to a variable, that you don't need to add the type information.